src/main/java/com/hypherionmc/modpublisher/tasks/GithubUploadTask.java

```diff
@@ -98,6 +99,9 @@ public void upload() throws Exception {
 
         UploadPreChecks.checkEmptyJar(extension, uploadFile, extension.getLoaders().get());
 
+        // We want to know if the existing release was a draft to avoid un-publishing releases
+        boolean wasDraft;
```

Comment:
This section of the file is starting to get messy. Not saying this should block the PR, just wanted to acknowledge that I'm part of the problem :joy: